docs/csharp/programming-guide/classes-and-structs/properties.md

```diff
@@ -74,6 +57,31 @@ Read-only properties can implement the `get` accessor as an expression-bodied me
 
 The `Name` property is a computed property. There's no backing field for `Name`. The property computes it each time.
 
+## Access control
+
+The preceding examples showed read / write properties. You can also create read-only properties, or give different accessibility to the set and get accessors. Suppose that your `Person` class should only enable changing the value of the `FirstName` property from other methods in that class. You could give the set accessor `private` accessibility instead of `public`:
```

Comment:
```suggestion
The preceding examples showed read / write properties. You can also create read-only properties, or give different accessibility to the set and get accessors. Suppose that your `Person` class should only enable changing the value of the `FirstName` property from other methods in the class. You could give the set accessor `private` accessibility instead of `internal` or `public`:
```